src/main/resources/application-dev.yml

```diff
@@ -44,6 +44,10 @@ hmpps:
     api:
       url: "https://hdc-api-dev.hmpps.service.justice.gov.uk"
 
+  cases:
+    last-minute:
+      prisons:
+
```

Comment:
No values for dev,.. as we get no results other wise very limited data in dev,.. run this locally with dev profile also